definitions/ext-deeper_connect/dashboard.json

```diff
@@ -0,0 +1,435 @@
+              }
+            ],
+            "thresholds": []
+          }
+        },
+        {
+          "visualization": {
+            "id": "viz.line"
+          },
+          "layout": {
+            "column": 1,
+            "row": 12,
+            "height": 3,
+            "width": 8
+          },
+          "title": "Balance",
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM DeeperConnectSample SELECT latest(balance.amount) as 'Balance Amount' TIMESERIES SINCE 1 week ago "
+              }
+            ],
+            "yAxisLeft": {
+              "zero": true
+            }
+          }
+        },
+        {
+          "visualization": {
+            "id": "viz.line"
+          },
+          "layout": {
+            "column": 9,
+            "row": 12,
+            "height": 3,
+            "width": 8
+          },
+          "title": "Credit Score",
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM DeeperConnectSample SELECT latest(balance.credit) as 'Credit Score' TIMESERIES SINCE 1 week ago "
+              }
+            ],
+            "yAxisLeft": {
+              "zero": true
+            }
+          }
+        },
+        {
+          "visualization": {
+            "id": "viz.line"
+          },
+          "layout": {
+            "column": 17,
+            "row": 12,
+            "height": 3,
+            "width": 8
+          },
+          "title": "Micropayment",
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM DeeperConnectSample SELECT latest(balance.channelAmount) as 'Micropayment Channel Amount' TIMESERIES SINCE 1 week ago "
+              }
+            ],
+            "yAxisLeft": {
+              "zero": true
+            }
+          }
+        },
+        {
+          "visualization": {
+            "id": "logger.log-table-widget"
+          },
+          "layout": {
+            "column": 1,
+            "row": 15,
+            "height": 3,
+            "width": 12
+          },
+          "title": "Sharing Logs",
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT `log_severity``timestamp``message` FROM Log WHERE `logtype` = 'deeper-sharing-log'"
```

Comment:
```suggestion
                "query": "SELECT `log_severity`,`timestamp`,`message` FROM Log WHERE `logtype` = 'deeper-sharing-log'"
```

Reply:
@cianBuckley so surprisingly to me also, this is actually correct this is the format the log widget wants as it is not a standard NRQL query.